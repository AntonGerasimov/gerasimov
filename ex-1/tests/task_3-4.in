15 16 32
