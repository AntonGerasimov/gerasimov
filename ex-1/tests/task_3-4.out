00000100 00001000 11110000 
